wolfssl/wolfcrypt/types.h

```diff
@@ -202,7 +202,11 @@ decouple library dependencies with standard string, memory and so on.
             #define WC_INLINE
         #endif
     #else
-        #define WC_INLINE
+        #ifdef __GNUC__
```

Comment:
Check for "inline" made in sp_int.h but not added here.

Reply:
Can't decipher what you're envisioning here.  Something to do with `SP_NOINLINE` defined in sp.h?

Reply:
Don't worry, confused NOINLINE with INLINE I think.